package.json

```diff
@@ -54,18 +54,18 @@
     "precommit": "yarn lint && yarn test"
   },
   "devDependencies": {
-    "@biomejs/biome": "=1.7.3",
-    "@napi-rs/cli": "^2.18.3",
+    "@biomejs/biome": "=1.8.3",
```

Comment:
we had. some issues last time we bumped biome. I'd rather have this done as it's own PR.

Reply:
The only issue I had was this option: `--apply-unsafe` has changed to `--write --unsafe`